test/runtests.jl

```diff
@@ -15,6 +15,12 @@ function testlu(A, MF, BF)
     nothing
 end
 testlu(A::Adjoint, MF::Adjoint, BF) = testlu(parent(A), parent(MF), BF)
+if isdefined(LinearAlgebra, :AdjointFactorization)
+    testlu(A::Adjoint, MF, BF) =
+        testlu(parent(A), parent(MF), BF)
+    testlu(A::Transpose, MF, BF) =
+        testlu(parent(A), parent(MF), BF)
+end
```

Comment:
```suggestion
testlu(A::Union{Transpose, Adjoint}, MF, BF) = testlu(parent(A), parent(MF), BF)
```

Reply:
🤦 Thanks! It took me a while to figure out what is passed where. Thought it was an easy change (and another... and another), and ended up with a mess. 🤣